prboom2/src/dsda/exdemo.c

```diff
@@ -402,6 +402,7 @@ void dsda_MergeExDemoFeatures(void) {
 static void DemoEx_GetFeatures(const wadinfo_t* header) {
   char* str;
   char signature[33];
+  long long value;
```

Comment:
For the features, given this is a very explicit set of 64 bit flags being written and read, I'm not as bothered by the prospect of using the rigorous PRI defines here. Let's actually try them out in this file and if I hate it I'll change it later 😄